app/views/me/notices/index.html.haml

```diff
@@ -1,12 +1,24 @@
-.h2.first=:notices.t
+%div
+  %span.h2.first=:notices.t
+  - if @notices.total_pages < 2 && @notices.any?
+    %span.h2.dissmiss-all-notices-button.dismiss-all-wrapper.pull-right
+      = dissmiss_all_notices_button
+
 - if @notices.empty?
   %ul.shaded
     %li= :none.t
 - else
-  = pagination_links @notices
+  %div
+    .pagination-wrapper.pull-left
+      = pagination_links @notices
+    - if @notices.total_pages > 1
```

Comment:
How about removing the upper pagination block? I don't see why we would need it. You scroll down to read all notices and then move on to the next page.